deployment/sriov-network-operator/values.yaml

```diff
@@ -25,9 +25,56 @@ operator:
   nameOverride: ""
   fullnameOverride: ""
   resourcePrefix: "openshift.io"
-  enableAdmissionController: false
   cniBinPath: "/opt/cni/bin"
   clusterType: "kubernetes"
+  admissionControllers:
+    enabled: false
+    certificates:
+      secretNames:
+        operator: "operator-webhook-cert"
+        injector: "network-resources-injector-cert"
+      certManager:
+        # When enabled, makes use of certificates managed by cert-manager.
+        enabled: false
+        # When enabled, certificates are generated via cert-manager and then name will match the name of the secrets
+        # defined above
+        generateSelfSigned: false
+      # If not specified, no secret is created and secrets with the names defined above are expected to exist in the
+      # cluster. In that case, the ca.crt must be base64 encoded twice since it ends up being an env variable.
+      custom:
+        enabled: false
+    #   operator:
+    #     caCrt: |
+    #       -----BEGIN CERTIFICATE-----
+    #       MIIMIICLDCCAdKgAwIBAgIBADAKBggqhkjOPQQDAjB9MQswCQYDVQQGEwJCRTEPMA0G
+    #       ...
+    #       -----END CERTIFICATE-----
+    #     tlsCrt: |
+    #       -----BEGIN CERTIFICATE-----
+    #       MIIMIICLDCCAdKgAwIBAgIBADAKBggqhkjOPQQDAjB9MQswCQYDVQQGEwJCRTEPMA0G
+    #       ...
+    #       -----END CERTIFICATE-----
+    #     tlsKey: |
+    #       -----BEGIN EC PRIVATE KEY-----
+    #       MHcl4wOuDwKQa+upc8GftXE2C//4mKANBC6It01gUaTIpo=
+    #       ...
+    #      -----END EC PRIVATE KEY-----
+    #   injector:
+    #     caCrt: |
```

Comment:
not related to this PR but was wondering:

kubelet supports creating tls secret but it only has cert and key.
if cert is self signed then it can be used for ca.

but in other cases, i wonder how ca ends up where it needed.

Reply:
> but in other cases, i wonder how ca ends up where it needed.

I guess you are referring on how cert-manager does it right? cert-manager will derive the CA out of the `Issuer` or `ClusterIssuer` accordingly. IIRC, if the `Issuer` is in the self signed mode then `ca.crt == tls.crt`. In other cases, it will get the CA (or chain) from the source (e.g. Vault, Let's Encrypt, whatever).

If you are talking about certificates in general, I've seen companies build their own PKI with multiple layers (i.e. multiple intermediate CAs) and have their own system of signing certificates. If a user is in such environment, I would expect that they populate the `ca.crt` field with the full chain of certs up until their leaf certificate (which is added in `tls.crt` and `tls.key` accordingly).

Reply:
about both :), thx for the insight.